### PR TITLE
BaSP-TG-91: Redux-based GET method for Employees

### PR DESCRIPTION
**Pull request by JuliMonte5**
Implemented redux on GET method for employees entity, the steps for this were:

- Create constants to get employees
- Create actions to get employees
- Create thunk to get employees
- Create employees reducer
- Connect list component with store to get employees

### CODE REVIEW

**Review by GonzaloAsencio — APPROVED** [review]

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -52,10 +46,23 @@ const Employees = () => {
     { heading: 'Actions' }
   ];
 
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+
+  if (error) {
```
Comment (GonzaloAsencio) [inline src/Components/Employees/index.js:53]: You can use Modal Info here!

Comment (JuliMonte5) [reply src/Components/Employees/index.js]: Added!

**Review by MaicolFreire001 — APPROVED** [review]
SGTM🚀

File: src/redux/employees/constant.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const GET_EMPLOYEES_PENDING = 'GET_EMPLOYEES_PENDING';
+export const GET_EMPLOYEES_FULFILLED = 'GET_EMPLOYEES_FULFILLED';
+export const GET_EMPLOYEES_REJECTED = 'GET_USERS_REJECTED';
```
Comment (ProDIGGY-vinilos) [inline src/redux/employees/constant.js]: You can change this to GET_EMPLOYEES_SUCCESS and GET_EMPLOYEES_ERROR to stay in armony to the whole team

Comment (JuliMonte5) [reply src/redux/employees/constant.js]: Solved

**Comment by GonzaloAsencio** [comment]
Resolve Conflict

**Review by fede230298 — APPROVED** [review]

**Pull request merged**